ui/decredmaterial/editor.go

```diff
@@ -268,21 +268,23 @@ func (e Editor) handleEvents() {
 }
 
 func (re RestoreEditor) Layout(gtx layout.Context) layout.Dimensions {
+	l := re.t.SeparatorVertical(int(gtx.Metric.PxPerDp)*re.height, int(gtx.Metric.PxPerDp)*2)
+	if re.Edit.Editor.Focused() {
+		re.TitleLabel.Color, re.LineColor, l.Color = re.t.Color.Primary, re.t.Color.Primary, re.t.Color.Primary
+	} else {
+		l.Color = re.t.Color.Gray1
+	}
 	border := widget.Border{Color: re.LineColor, CornerRadius: values.MarginPadding8, Width: values.MarginPadding2}
 	return border.Layout(gtx, func(gtx C) D {
 		return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
 			layout.Rigid(func(gtx C) D {
-				return layout.Inset{
-					Left:  unit.Dp(10),
-					Right: unit.Dp(10),
-				}.Layout(gtx, func(gtx C) D {
+				gtx.Constraints.Min.X = int(gtx.Metric.PxPerDp) * 40
```

Comment:
replace the explicit DPI conversion with the standard unit type as done in PR #418